### DISCUSSION TITLE
Freedom

### DISCUSSION THREAD

**Discussion by marshallsoft** [ImageMagick/ImageMagick#4276] [Kudos and Rants]
How do you give this away for free?  This is badass.  Y'all must be doing something else to make money... because damn.  I love you.

**Comment by urban-warrior** [comment]
A tenet we employ is "for any given situation, always give back more than you take."  An open-source ImageMagick meets and exceeds this core value.

**Comment by marshallsoft** [comment]
Then, all I can say is thank you.

________________________________
